### PR TITLE
Fix iOS project

### PR DESCRIPTION
**Pull request by olilarkin**
For some reason the old iPlug2OOS template would not build to real devices only the simulator. This fixes that

### CODE REVIEW

File: NeuralAmpModeler/NeuralAmpModeler.xcworkspace/contents.xcworkspacedata [hunk 1]
```diff
@@ -1,12 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Workspace
    version = "1.0">
-   <FileRef
```
Comment (sdatkinson) [inline NeuralAmpModeler/NeuralAmpModeler.xcworkspace/contents.xcworkspacedata:4]: This looks like a problem? You're removing these source files from the project?

Comment (sdatkinson) [reply NeuralAmpModeler/NeuralAmpModeler.xcworkspace/contents.xcworkspacedata:4]: Apologies for the premature comment while I refresh myself on what each button does on GitHub 😅

Comment (olilarkin) [reply NeuralAmpModeler/NeuralAmpModeler.xcworkspace/contents.xcworkspacedata:4]: I removed them from the xcode workspace... they should be in the xcode projects individually

File: NeuralAmpModeler/projects/NeuralAmpModeler-iOS.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -21,70 +21,76 @@
 /* End PBXAggregateTarget section */
 
 /* Begin PBXBuildFile section */
-		4F0D965B23099F5A00BFDED0 /* IPlugProcessor.cpp in Sources */ = {isa = PBXBuildFile; fileRef = 4FFF103420A0E55900D3092F /* IPlugProcessor.cpp */; };
```
Comment (sdatkinson) [inline NeuralAmpModeler/projects/NeuralAmpModeler-iOS.xcodeproj/project.pbxproj:24]: Can you summarize these changes? Looks like just reorganizing? Or were any added/removed?

File: NeuralAmpModeler/projects/NeuralAmpModeler-iOS.xcodeproj/xcshareddata/xcschemes/iOS-Update plists.xcscheme [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Scheme
-   LastUpgradeVersion = "1420"
```
Comment (sdatkinson) [inline NeuralAmpModeler/projects/NeuralAmpModeler-iOS.xcodeproj/xcshareddata/xcschemes/iOS-Update plists.xcscheme:3]: Looking at this, I feel like maybe a change I had done moved these versions forward. Are we going to be tug-of-war'ing these lines, or is there a better way and/or am I just misremembering?

**Review by sdatkinson — COMMENTED** [review]
A few questions I just need answers to. May be fine (I haven't pulled it down and tried to use it.

File: NeuralAmpModeler/projects/NeuralAmpModeler-iOS.xcodeproj/project.pbxproj (hunk 2 above)
Comment (olilarkin) [reply NeuralAmpModeler/projects/NeuralAmpModeler-iOS.xcodeproj/project.pbxproj:24]: life is too short! i'm fixing the iOS project. This involves reduplicating the iPlug2OOS template and bringing in the modified .xcodeproj and adding the files, getting it to build

File: NeuralAmpModeler/projects/NeuralAmpModeler-iOS.xcodeproj/xcshareddata/xcschemes/iOS-Update plists.xcscheme (hunk 3 above)
Comment (olilarkin) [reply NeuralAmpModeler/projects/NeuralAmpModeler-iOS.xcodeproj/xcshareddata/xcschemes/iOS-Update plists.xcscheme:3]: These are just because i am using the latest xcode. It's irrelevant really just means xcode wont give suggestions to upgrade the project

Comment (olilarkin) [reply NeuralAmpModeler/projects/NeuralAmpModeler-iOS.xcodeproj/xcshareddata/xcschemes/iOS-Update plists.xcscheme:3]: sorry other way round. I guess the template project is not upgraded for xc 1420

Comment (olilarkin) [reply NeuralAmpModeler/projects/NeuralAmpModeler-iOS.xcodeproj/xcshareddata/xcschemes/iOS-Update plists.xcscheme]: reverted those plist scheme changes

File: NeuralAmpModeler/projects/NeuralAmpModeler-iOS.xcodeproj/project.pbxproj (hunk 2 above)
Comment (sdatkinson) [reply NeuralAmpModeler/projects/NeuralAmpModeler-iOS.xcodeproj/project.pbxproj:24]: Sounds good 😅 Just a dense diff of configgy code that on first scan didn't look like it was doing as much as the red & green implied; figured a quick answer from you would be the fastest way to get this ok'd 🙂

File: NeuralAmpModeler/NeuralAmpModeler.cpp [hunk 4]
```diff
@@ -204,82 +210,86 @@ NeuralAmpModeler::NeuralAmpModeler(const InstanceInfo &info)
...
-          ss << "Failed to load NAM model. Message:\n\n"
-             << msg << "\n\n"
-             << "If the model is an old \"directory-style\" model, it can be "
-                "converted using the utility at "
-                "https://github.com/sdatkinson/nam-model-utility";
-          pGraphics->ShowMessageBox(ss.str().c_str(), "Failed to load model!",
-                                    kMB_OK);
+      WDL_String initFileName;
+      WDL_String initPath(this->mNAMPath.remove_filepart());
+      pGraphics->PromptForFile(initFileName, initPath, EFileAction::Open, "nam",
```
Comment (sdatkinson) [inline NeuralAmpModeler/NeuralAmpModeler.cpp:215]: Oh thank you! ❤️

File: NeuralAmpModeler/NeuralAmpModeler.cpp [hunk 5]
```diff
@@ -204,82 +204,86 @@ NeuralAmpModeler::NeuralAmpModeler(const InstanceInfo &info)
...
```
Comment (olilarkin) [inline NeuralAmpModeler/NeuralAmpModeler.cpp:215]: @sdatkinson here are the changes for filtering "nam" extensions

**Review by sdatkinson — APPROVED** [review]
LGTM! 👍🏻

**Comment by sdatkinson** [comment]
Resolves #96
